src/main/java/Company.java

```diff
@@ -0,0 +1,92 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class Company {
+    private Employee employee;
+    private static final String IT_DEPARTMENT = "IT";
+    private static final String SUPPORT_DEPARTMENT = "Support";
+    private static final String MANAGEMENT_DEPARTMENT = "Management";
+
+    static Employee[] readFile(File file) throws FileNotFoundException, NumberFormatException {
+        Scanner scanner = new Scanner(file);
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY